### PR TITLE
Feature/MD-1576 Initial Review *DO NOT MERGE*

### PR DESCRIPTION
**Pull request by elizabethmidgett**
Initial review to go along with core PR. Please do not merge until Core is ready to go.

### CODE REVIEW

**Comment by larastocks** [comment]
+1

File: apiary.apib [hunk 1]
```diff
@@ -445,6 +445,48 @@ Retrieve all Business Asset Attribute Types for a specified Business Asset Type
 
 + Response 204
 
+# Group Manufacturers
+
+## Manufacturer Collection [/accounts/{accountId}/manufacturers{?page,pagesize,orderby,purpose,includeAudit}]
++ Parameters
```
Comment (aoffpo) [inline apiary.apib:453]: Per our standards, we need a description here and after any ##

Comment (elizabethmidgett) [inline apiary.apib:453]: Updated for the items in the scope of this story; however, the majority of the collections are missing this documentation.

**Comment by aoffpo** [comment]
+1

**Pull request merged**